drivers/xen/xenbus.c

```diff
@@ -1,21 +1,20 @@
-/*
- ****************************************************************************
+/* SPDX-License-Identifier: GPL-2.0+
+ *
  * (C) 2006 - Cambridge University
  * (C) 2020 - EPAM Systems Inc.
- ****************************************************************************
  *
- *		File: xenbus.c
- *	  Author: Steven Smith (sos22@cam.ac.uk)
- *	 Changes: Grzegorz Milos (gm281@cam.ac.uk)
- *	 Changes: John D. Ramsdell
+ * File: xenbus.c
+ * Author: Steven Smith (sos22@cam.ac.uk)
+ * Changes: Grzegorz Milos (gm281@cam.ac.uk)
+ * Changes: John D. Ramsdell
  *
- *		Date: Jun 2006, chages Aug 2005
+ * Date: Jun 2006, changes Aug 2006
  *
- * Environment: Xen Minimal OS
+ * Environment: Xen Minimal OS [1]
  * Description: Minimal implementation of xenbus
  *
- ****************************************************************************
- **/
+ * [1] - https://wiki.xenproject.org/wiki/Mini-OS
```

Comment:
"Better provide a link to the original source." might mean that we can have a link to the source repository